lib/CoreBot.js

```diff
@@ -126,6 +126,18 @@ function Botkit(configuration) {
         this.say = function(message) {
             this.addMessage(message);
         };
+        
+        this.react = function(reaction) {
+            bot.api.reactions.add({
+                timestamp: this.transcript[this.transcript.length-1].ts,
+                channel: this.source_message.channel,
+                name: reaction
+            },function(err, res) {
```

Comment:
Did you use the linter? 

According to linter there should be an empty space between `,` and `function`

Please test your code using `npm test`